backend/src/test/java/sonia/meetapp/app/integration/IntegrationTest.java

```diff
@@ -258,6 +290,30 @@ void createAccountEmailAlreadyExists() throws Exception {
                  """).with(csrf())).andExpect(status().is(403));
     }
 
+    @DirtiesContext
+    @Test
+    void createAccountPasswordDoesNotMatch() throws Exception {
+        mockMvc.perform(post("/hello").contentType(APPLICATION_JSON).content("""
+                {"email": "abcde@gmail.com",
+                               "password": "aaaaaa",
+                               "repeatPassword":"bbbbbb",
```

Comment:
hier ist kein Leerzeichen nach dem Doppelpunkt, in den Zeilen darüber schon :D, ich find es mit schöner.